ROLES.md

```diff
@@ -242,7 +274,89 @@ The following apply to the area / component for which one would be an owner.
+
+- Mentoring and guiding approvers, members, and new contributors.
+
+## Working Group Execution Lead
+
+Working group execution leads, or just ‘execution leads’, are responsible for
+the overall health and execution of the working group itself. Execution leads
+work with tech leads to ensure that the working group is making progress toward
+its goals, is aligned with the project roadmap, etc. The execution lead may also
+be the tech lead in a smaller working group, but they are distinct roles.
+
+### Requirements
+
+- Participant in the working group for at least 3 months, for example as scribe
+  or approver.
+
+- Recognized as having expertise in the group’s subject matter.
+
+- Sponsored by the technical oversight committee.
+
+### Responsibilities and privileges
+
+- Run their working group as explained in the
+  [Working Group Processes](./mechanics/WORKING-GROUP-PROCESSES.md).
+
+  - Meetings. Prepare the agenda and run the regular working group meetings.
+
+  - Notes. Ensure that meeting notes are kept up to date. Provide a link to the
+    recorded meeting in the notes. The lead may delegate note-taking duties to
+    the scribe.
+
+  - Roadmap. Establish and maintain a roadmap for the working group outlining
+    the areas of focus for the working group over the next 3 months.
+
+  - Report. Report current status to the TOC meeting every 6 weeks.
+
+- Holistic responsibility for their working group's [feature
+  tracks](./mechanics/FEATURE-TRACKS.md): tracking, health, and execution.
+
+- Perform issue triage on GitHub.
+
+- Apply/remove/create/delete GitHub labels and milestones.
+
+- Write access to repo (assign issues/PRs, add/remove labels and milestones,
+  edit issues and PRs, edit wiki, create/delete labels and milestones).
+
+- Expected to work to holistically maintain the health of the working through:
+
+  - Being a good role model
+
+  - Be an advocate for the working group inside and outside of the community
+
+  - Foster a welcoming and collegial environment
+
+  - Mentoring and guiding approvers, members, and new contributors.
+
+## Scribe
+
+One of the most underrated roles in open source projects is the role of note
+taker. The importance and value of this role is frequently overlooked and
+underestimated. Since one of the core project values is transparency, we have an
+explicit scribe role to recognize these types of contributions. Working group
+scribes assist the Working Group leads with the mechanical processes around
+Working Group meetings.
+
+### Requirements
+
+- Participant in the working group for at least 1 month.
+
+- Pattern of attendance and note-taking during working group meetings and one-offs.
+
+- Sponsored by a working group execution or technical lead.
+
+### Responsibilities and privileges
+
+- Attend working group meetings and one-offs whenever possible.
+
+- Ensure that important information from meetings makes it into the WG notes.
+
+- Post WG recordings to the team drive.
```

Comment:
There are technical limitations here today.

Reply:
Do you think they are surmountable? Should we strike this? Will ping you about this one offline

Reply:
Depends on how stuff like Zoom manifests I suppose, I don't know enough about the alternatives, but I don't think they are surmountable with the system we have now due to google.com org.